utils/cli/core/core.py

```diff
@@ -69,7 +71,22 @@ def tasks_create(self, name, labels, bug, resource_type, resources, **kwargs):
         response.raise_for_status()
         response_json = response.json()
         log.info('Created task ID: {id} NAME: {name}'.format(**response_json))
-        self.tasks_data(response_json['id'], resource_type, resources)
+        task_id = response_json['id']
+        self.tasks_data(task_id, resource_type, resources)
+
+        if annotation_path != '':
+            url = self.api.tasks_id(task_id)
+            task_size = self.session.get(url).json()['size']
+
+            log.info('Awaiting data compression before uploading annotations...')
+
+            while task_size == 0:
+                sleep(cooldown_period_in_secs)
+                task_size = self.session.get(url).json()['size']
```

Comment:
It would be better to make a request at the endpoint `/api/v1/tasks/{task_id}/status` to get the status, otherwise, if the task creation process fails, you will always get `size == 0`
For the task that are still creating you get smth like:
```
{state: "Started", message: "Images are being compressed 90%"}
```
For the successfully created task it should be:
```
{"state":"Finished","message":""}
```

Reply:
Hey @azhavoro, thanks for the requests. Thanks for the idea on the status request.

I fixed the test error and and changed the request verification as you asked. Every test is passing now.